static/docs/user-guide/introduction.md

```diff
@@ -0,0 +1,32 @@
+# Basic Concepts
+
+The DVC user guide constitutes DVC files and directories, DVC file format,
+external dependencies, external outputs, update a tracked file, anonymized usage
+analytics which are part of basic. Customizations has DVC autocomplete, plugins,
+developement version and contributing guide.
+
+Let's understand the basic concepts as you use dvc.
+
+## Workspace
+
+Work space is the current work directory.
+
+## Cache
+
+DVC cache is a hidden (by default it's located in the `.dvc/cache` directory)
+storage. For files that are under DVC control it keeps them and their different
+versions. You can find more information on cache as described in this [doc](https://dvc.org/doc/commands-reference/config#cache).
+
+## Remote
+
+In DVC you can have remote data storage or a remote git hosted code server.
+A remote can be specified by the remote type prefix and a path. You can find
```

Comment:
`remote type prefix` - remote is usually specified by its URL

Reply:
I think in dvc adding a remote should be specified by type + prefix? Remotes can be
- `local`
- `s3`
- `gs`
- `azure`
- `ssh`
- `hdfs`
- `http`

You would run `dvc remote add -d /typer/remote path/url`.